lib/apiservers/engine/backends/container.go

```diff
@@ -915,13 +917,18 @@ func (c *Container) ContainerInspect(name string, size bool, version version.Ver
 		}
 	}
 
-	inspectJSON, err := containerInfoToDockerContainerInspect(name, results.Payload)
+	inspectJSON, err := containerInfoToDockerContainerInspect(vc, results.Payload)
```

Comment:
I added a check near the top of the function where we get the container.  It doesn't make sense for us to get down here if vc was nil.